src/main/java/frc/robot/Robot.java

```diff
@@ -142,6 +142,58 @@ public void robotInit() {
                 }, Set.of(armTelescopicSystem, clawJointSystem, clawGripperSystem)
         );
 
+
+        Command collectFromSourceCommand = new ParallelCommandGroup(
```

Comment:
All the commands are legit almost exactly the same, just different positions. Create a method

`createCommandGroupSimple(armLength, armAngle, clawAngle)` which creates the command group from these parameters and use that method to create all the commands with different parameters.